supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorJobExecutionActor.scala

```diff
@@ -266,7 +266,7 @@ class HtCondorJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   private def modifyCommandForDocker(jobCmd: Try[String], localizedInputs: CallInputs): Try[String] = {
     Try {
       val dockerInputDataVol = localizedInputs.collect {
-        case (k, v) => v match {
+        case (k, v) if v.wdlType == WdlFileType || v.wdlType == WdlArrayType(WdlFileType) => v match {
```

Comment:
...and actually this can just do `localizedInputs.values.collect...` and just leave the keys out of it. 😄